pkg/policy/opa/engine.go

```diff
@@ -46,13 +46,13 @@ var (
 )
 
 // NewEngine returns a new OPA policy engine
-func NewEngine(policyPath string) (*Engine, error) {
+func NewEngine(policyPath string, scanRules, skipRules []string) (*Engine, error) {
```

Comment:
Wondering if `scanRules` and `skipRules` can become part of the `Engine{}` struct. 

Would like an opinion from @williepaul on this.

Reply:
Added a function FilterRules(string, []string, []string) to Engine interface and moved the Init function out of the NewEngine function.